doc/commands/cmd_report.rst

```diff
@@ -104,6 +104,19 @@ inclusive are missing coverage.  Missed branches are shown with an arrow:
 backwards in a file, so you might see a branch from a later line to an earlier
 line, like "55->50".
 
+
+Missing branches
+----------------
+
+Missing branches are shown such as ``12->10``.why is it backward?
```

Comment:
This make it sound like branches are always reported in reverse. We should be clear that it's only sometimes.